src/components/Filters/story.js

```diff
@@ -241,6 +241,16 @@ storiesOf('Next/Filters', module)
       </Box>
     )
   })
+  .add('Compact', () => {
+    const props = {
```

Comment:
Why isn't this inline with the component?

Reply:
changed to inline as recommended